### PR TITLE
chore: update renovate config

### PR DESCRIPTION
**Pull request by esoubiran-aneo**
You can find an example: https://github.com/esoubiran-aneo/ArmoniK.Samples

### CODE REVIEW

**Review by aneojgurhem — APPROVED** [review]

**Comment by esoubiran-aneo** [comment]
Can't merge this PR @aneojgurhem (do not have write access).

**Pull request merged**
